crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -71,15 +71,15 @@ impl AdminInterfaceApi {
                     .await?;
                 Ok(AdminResponse::AdminInterfacesAdded)
             }
-            GetDnaDefinition(dna_hash) => {
+            GetDnaDefinition(cell_id) => {
                 let dna_def = self
                     .conductor_handle
-                    .get_dna_def(&dna_hash)
-                    .ok_or(ConductorApiError::DnaMissing(*dna_hash))?;
+                    .get_dna_def(&cell_id)
+                    .ok_or(ConductorApiError::CellMissing(*cell_id))?;
                 Ok(AdminResponse::DnaDefinitionReturned(dna_def))
             }
             UpdateCoordinators(payload) => {
-                let UpdateCoordinatorsPayload { dna_hash, source } = *payload;
+                let UpdateCoordinatorsPayload { cell_id, source } = *payload;
```

Comment:
Also here, we need to break the admin API since `dna_hash` is not an unambiguous specifier as to for which cell to update coordinators.